src/main/kotlin/org/kamiblue/client/module/modules/render/NewChunks.kt

```diff
@@ -33,6 +33,7 @@ internal object NewChunks : Module(
     category = Category.RENDER
 ) {
     private val relative by setting("Relative", false, description = "Renders the chunks at relative Y level to player")
+    val renderMode = setting("RenderMode", RenderMode.BOTH)
```

Comment:
I would say getting all these out of NewChunks to avoid unnecessary usages across features since it can make codes harder to maintain.

Reply:
Should I add an option to NewChunks instead that lets you disable rendering in the world? That would be a weird setting, for someone who doesn't know that NewChunks can be shown in the radar.

Reply:
I think the usage here is fine, a comment both RenderMode in NewChunks.kt and in the Radar.kt linking them might be in order.

Alternatively, if we want to keep all functionality inside of the feature/module, then a new "RenderRadarEvent" should be created that Radar sends to the event system that requests things to be drawn on the radar section. This way features will stay within their own class and can be sorted by priority for what goes on top and bottom.

Reply:
@Xiaro what do you think about nopjmps suggestion? If you think it's ok I would just add a comment explaining where the setting is used.

Reply:
I'd prefer an event based way, yes.

Reply:
I'll add an event then, although I'm not sure how the kami blue event system works. So it might take a while until I have time to look into it.

Reply:
I just pushed a Event based solution, which is expandable for the future. It would be good, if someone can check, if I used the correct event and listener types.

Reply:
I'll give it a review tomorrow, going to sleep soon. Thank you!